docs/specifications/tests/Basic-TP/basic00.md

```diff
@@ -6,25 +6,45 @@
 ### Objective
 
 In order to begin testing the domain name from the input must be a valid
-domain name. The domain name must follow the rules defined in section 2.1
-of [RFC 1123](https://tools.ietf.org/html/rfc1123#section-2.1). The
-objective of this test is to see if it is possible to have the domain name
+domain name. The domain name must either be a 
+
+1. a valid IDN name (Internationalized Domain Name) ([RFC 5890](https://tools.ietf.org/html/rfc5890#page-13), 
+section 2.3.2.3), or
+2. a valid ASCII domain.
+
+The ASCII domain name is valid if follows the rules defined in section 2.1
+of [RFC 1123](https://tools.ietf.org/html/rfc1123#section-2.1), i.e. only
+consists of the ASCII characters "a-z", "A-Z", "0-9", "." and "-" with the
+extension of "_", standardized for SRV records 
+([RFC 2782](https://tools.ietf.org/html/rfc2782)).
+
+A valid IDN name with non-ASCII code points can always be converted 
+to a valid ASCII domain meeting the requirement above.
+
+The objective of this test is to see if it is possible to have the domain name
 mapped into a DNS packet in order to proceed with further testing.
 
+
 ### Inputs
 
-The label of the domain name to be tested.
+The domain name to be tested.
 
 ### Ordered description of steps to be taken to execute the test case
 
-1. Remove any trailing dot from the input domain name.
-2. If the total length of the input is more than 253 octets, this test
+1. If the domain name has more than one tailing dot, the test case
```

Comment:
You are right that step 1 partly covers what step 8 covers. Maybe it is better to change step 1 to "If the input domain name contains two or more consecutive dots, then this test case fails" and remove step 8. Then that simple error will be caught early.